tests/WaveFileAudioInputStream.ts

```diff
@@ -19,7 +19,7 @@ export class WaveFileAudioInput {
     public static LoadArrayFromFile(filename: string): ArrayBuffer {
         const fileContents: Buffer = fs.readFileSync(filename);
 
-        const ret = Uint8Array.from(fileContents);
+        const ret = Uint8Array.from(fileContents.slice(44));
```

Comment:
Some wav headers are not 44 bytes long e.g. more than 2 channels pcm file normally has more than 44 bytes. Please add a comment saying for now only support 1 channel pcm data as wav.